source/_integrations/bluetooth.markdown

```diff
@@ -87,6 +87,14 @@ The following methods are known to work to add multiple adapters:
 
 Integrations that have followed the [Best practices for library authors](https://developers.home-assistant.io/docs/network_discovery?_highlight=bluetooth#best-practices-for-library-authors) will automatically connect via the adapter with the best signal and failover to an active adapter if one becomes unavailable.
 
+## Passive Scanning
```

Comment:
On the code PR you provide good guidance on active vs passive: https://github.com/home-assistant/core/pull/75542#issuecomment-1225876035

It might be good to provide similar guidance here

Reply:
I intentionally didn't want to encourage passive scanning because it will lead to more issues in the queue.